### ISSUE TITLE
hyperparameter for few-shot classification in ImageNet 

### ISSUE THREAD

**Issue by ChoiDae1** [locuslab/FLYP#13]
Hello! I read your paper very well.
I have a question about your finetuning detail.
In paper's  ```figure3```,  there are results about few-shot classification on ImageNet. (4, 16, 32 shot)
To reproduce FLYP results, how can I set finetuning parameter. (ex. learning rate, batch size, warmup_length, weight decay.. etc)

**Comment by SachinG007** [comment]
Thanks for your interest in our work.
So for our gridsearch in few-shot learning experiments, we create a validation set of the same size as the train set (chosen randomly). And we repeat this process 50 times due to the noise involved because of low number of samples (See Section 4, experiment protocol). 
I set warmup_length as 0 for fewshot learning, as very small number of steps are involved. I kept batch size as 256, weight_decay as 0 and tuned learning rate based on the gridsearch process noted above. 

I have provided the sample scripts for SST2 and PCam few-shot learning experiments in the README. 

Thanks
